webservices/common/models/itemized.py

```diff
@@ -26,6 +30,30 @@ def memoed_subtotal(self):
         return self.memo_code == 'X'
 
 
+class BaseRawItemized(db.Model):
```

Comment:
I'd say that these new tables are different enough from the regular itemized resources to warrant a new base class (mostly because the processed itemized base class has more columns).  And I at least don't know of a way to use a schema base class that won't error out in SqlAlchemy given our current configuration, may be a way but not sure.

And yes, the new base class will be used for other raw itemized resources (a,b), all the columns in the base class are in common across the e-file tables.